homeassistant/components/heos/quality_scale.yaml

```diff
@@ -5,7 +5,7 @@ rules:
     status: done
     comment: Integration is a local push integration
   brands: done
-  common-modules: todo
+  common-modules: done
```

Comment:
Side note: I don't think this PR was a requirement to complete this quality scale rule.